bartop-ui/.env.example

```diff
@@ -4,5 +4,4 @@ REACT_APP_AUTH0_CLAIM_NAMESPACE={{ the namespace used in the auth0 rules for acc
 
 REACT_APP_AUTH0_BARTOP_API_AUDIENCE={{ the audience string for the bartop API through Auth0 }}
 
-REACT_APP_URL={{ the url that the app is running on, usually http://localhost:3000 for local development }}
```

Comment:
The UI now determines this dynamically so we don't need it as an environment variable

Reply:
fancy boi